plugins/linux-sleep/fu-plugin-linux-sleep.c

```diff
@@ -32,6 +32,7 @@ fu_plugin_linux_sleep_add_security_attrs(FuPlugin *plugin, FuSecurityAttrs *attr
 	}
 	if (g_strstr_len(buf, bufsz, "[deep]") != NULL) {
 		fwupd_security_attr_set_result(attr, FWUPD_SECURITY_ATTR_RESULT_ENABLED);
+		fwupd_security_attr_add_flag(attr, FWUPD_SECURITY_ATTR_FLAG_ACTION_CONFIG_FW);
```

Comment:
It's config OS too in some cases.  If the system offers both and you changed the default on the kernel command line or in sysfs for example.

Reply:
Fixed.